task-maker-exec/src/lib.rs

```diff
@@ -105,17 +104,11 @@ mod scheduler;
 mod worker;
 mod worker_manager;
 
+const KEY_MATERIAL: &'static str = "task-maker-rust 99e4e7117fbf4ed6f8f1850c6acb61b6";
```

Comment:
What is this hex?

Reply:
The [documentation](https://docs.rs/blake3/latest/blake3/fn.derive_key.html) says that value must be: _"hardcoded, globally unique, and application-specific"_, so I added a randomly generated hex string.